Expensify/App
# [Hold for n6] Updated title for magic link screen

@thienlnam Can you please merge this?

### Details
- Updated the title on the magic link screen

### Fixed Issues

$ https://github.com/Expensify/App/issues/5234

### Tests
1. Verified the title on Forgot Password/Magic link screen
2. Changed the language to see the Spanish translation



### QA Steps
1. Go to the login page and enter your login
2. Click on the "Forgot?" link
3. Verify the bold title of the page with the new copy


### Tested On


### Screenshots


#### Web

<img width="374" alt="Screenshot 2021-09-16 at 1 38 48 AM" src="https://user-images.githubusercontent.com/3069065/133503495-63b4ed70-52a5-4733-ba85-589913a89b7d.png">


#### Mobile Web

<img width="333" alt="Screenshot 2021-09-16 at 1 39 04 AM" src="https://user-images.githubusercontent.com/3069065/133503289-de832110-1119-4a47-9f4c-ce6fa3c41c73.png">


#### Desktop

<img width="434" alt="Screenshot 2021-09-16 at 1 47 45 AM" src="https://user-images.githubusercontent.com/3069065/133503471-9fe67a08-9711-4014-b540-8f09cfefdde2.png">

#### iOS

<img width="353" alt="Screenshot 2021-09-16 at 1 42 44 AM" src="https://user-images.githubusercontent.com/3069065/133503354-5f304c6c-4453-40c6-bda2-b41a6df29d8a.png">

#### Android

<img width="326" alt="Screenshot 2021-09-16 at 1 40 02 AM" src="https://user-images.githubusercontent.com/3069065/133503329-faf40b73-d6ac-4315-8c25-104ab8dc9275.png">
